--- /root/raft.go	2000-01-01 02:36:56.251633770 +0000
+++ etcdserver/raft.go	2000-01-01 02:46:26.296634041 +0000
@@ -37,6 +37,11 @@ type RaftTimer interface {
 }
 
 type raftNode struct {
+	// Cache of the latest raft index and raft term the server has seen
+	index uint64
+	term  uint64
+	lead  uint64
+
 	raft.Node
 
 	// config
@@ -51,11 +56,6 @@ type raftNode struct {
 	// clients should timeout and reissue their messages.
 	// If transport is nil, server will panic.
 	transport rafthttp.Transporter
-
-	// Cache of the latest raft index and raft term the server has seen
-	index uint64
-	term  uint64
-	lead  uint64
 }
 
 // for testing
