--- ../watcher_hub.go	2015-03-09 22:33:04.542361131 -0400
+++ gopath/src/github.com/coreos/etcd/store/watcher_hub.go	2015-03-09 22:33:30.352229472 -0400
@@ -31,9 +31,9 @@
 // event happens between the end of the first watch command and the start
 // of the second command.
 type watcherHub struct {
+	count        int64 // current number of watchers.
 	mutex        sync.Mutex
 	watchers     map[string]*list.List
-	count        int64 // current number of watchers.
 	EventHistory *EventHistory
 }
 
